src/gravity/Artist.ts

```diff
@@ -25,7 +24,12 @@ export const EmbeddedArtist = Record({
   forsale_artworks_count: Number,
   hometown: String.Or(Undefined),
   image_url: String.Or(Null),
-  image_urls: Dictionary(String, "string"), // TODO: Flesh out expected keys
+  image_urls: Record({
+    square: String.Or(Null).Or(Undefined),
+    four_thirds: String.Or(Null).Or(Undefined),
+    tall: String.Or(Null).Or(Undefined),
+    large: String.Or(Null).Or(Undefined),
+  }),
```

Comment:
@mzikherman thought this was interesting: in looking, I could find Artists that have individually nullable fields on their image_urls objects OR just missing keys (undefined). Whereas Artworks always just have missing keys, never nulls (at least I haven't found any yet in practice).